code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -101,8 +101,8 @@
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 2, "pasta" = 1)
 
 /obj/item/reagent_containers/food/snacks/macpizzaslice
-	name = "mac n cheese pizza slice"
-	desc = "A delicious slice of pizza topped with macaroni cheese... wait, what the hell? Who would do this?!"
+	name = "Macaroni & Cheese pizza slice"
```

Comment:
Ah, wasnt sure about that since half the food is lower case, half is upper case. Maybe I'll take a pass to make that consistent if that's the standard